packages/form-builder/src/common/getWindowData.ts

```diff
@@ -13,6 +22,8 @@ declare global {
                 isInstalled: boolean;
             },
             gatewaySettingsUrl: string;
+            templateTags: TemplateTag[];
```

Comment:
I think we should rename this to be specific to emails like `emailTemplateTags` - reason being, there are other settings like the thank you page that have their own.

Reply:
Good point - I'll refactor the name.